### PR TITLE
Revised English captions practicum04

### PR DESCRIPTION
**Pull request by mvcaro**
Complete revision of English captions for practicum04

### CODE REVIEW

**Comment by xcastilla** [comment]
I'll review this one.

File: docs/en/week04/practicum04.sbv [hunk 1]
```diff
@@ -1,379 +1,379 @@
...
 
 0:00:28.529,0:00:32.640
-kind of you know fixing the error in
+kind of, you know, fixing the error in
 English in the English grammar or
 
 0:00:32.640,0:00:37.290
-whatever so if you if you're interested
+whatever, so if you if you're interested
 in a help in improving the content of
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:31]:
```suggestion
in helping improving the content of
```

File: docs/en/week04/practicum04.sbv [hunk 2]
```diff
@@ -1,379 +1,379 @@
...
+First time, so you know, I'm a little bit
 
 0:01:03.930,0:01:11.250
-tense last time I screw up with a
-notebook so okay alright so we're gonna
+tense. Last time I screw up with a
+notebook, so okay, alright. So we're gonna
 
 0:01:11.250,0:01:15.659
 be talking starting with a small review
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:58]:
```suggestion
be starting with a small review
```

File: docs/en/week04/practicum04.sbv [hunk 3]
```diff
@@ -1,379 +1,379 @@
...
 
 0:01:51.170,0:02:06.450
-okay am i waiting a little bit let me
-wait for a sec ready yes no shake your
+Okay. Am I waiting a little bit? Let me
+wait for a sec. Ready?? Yes? No? shake your
 
 0:02:06.450,0:02:12.150
-head okay fantastic alright so we were
+head. Okay fantastic, alright so were
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:86]:
```suggestion
head. Okay fantastic, alright so we were
```

File: docs/en/week04/practicum04.sbv [hunk 4]
```diff
@@ -1,379 +1,379 @@
...
+and you have as many columns as the
+dimension where you're shooting from, ok?
 
 0:03:34.200,0:03:39.930
-all right so let's expand this one right
+All right so let's expand this one, 
 so this matrix here is gonna be equal to
 
 0:03:39.930,0:03:49.549
 what do you have a 1 1 a 1 2 so on until
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:154]: @Atcold What do you think about  the caption for subscripts? Should we leave it like this or perhaps change it to something more symbolic like a_(1,1)   a_(1,2) etc.

File: docs/en/week04/practicum04.sbv [hunk 5]
```diff
@@ -1,379 +1,379 @@
...
 0:05:12.409,0:05:15.800
 here I'm gonna be multiplying this by a
-column vector right so my column vector
+column vector, right? so my column vector
 
 0:05:15.800,0:05:23.409
-I'm gonna be write it like this alright
-so what is the output of this operation
+I'm gonna be write it like this.
+So what is the output of this operation?
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:202]:
```suggestion
Alright, so what is the output of this operation?
```

File: docs/en/week04/practicum04.sbv [hunk 6]
```diff
@@ -1,379 +1,379 @@
...
 here so I'd like to figure out what is
 
 0:07:00.140,0:07:10.970
-the meaning of writing a T times X right
-so my a T is gonna be my generic a I so
+the meaning of writing a T times x right
+so my a T is gonna be my generic a i so
 
 0:07:10.970,0:07:24.680
 let's assume in this case when n equal
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:272]:
```suggestion
let's assume in this case when n equals 2
```

File: docs/en/week04/practicum04.sbv [hunk 7]
```diff
@@ -1,379 +1,379 @@
...
-to okay so what is ATX so ATX is going
+to okay so what is a T x? so a T x is going
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:273]:
```suggestion
okay? so what is a T x? so a T x is going
```

File: docs/en/week04/practicum04.sbv [hunk 8]
```diff
@@ -1,379 +1,379 @@
...
 0:07:37.720,0:07:48.620
 then here you have like it's gonna be my
-X and is this gonna be here my X I so
+x and is this gonna be here my xi, so
 
 0:07:48.620,0:07:57.430
-what is this what is the output of this
-product here these are here
+what is the output of this
+product here? these one here.
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:289]:
```suggestion
product here? This one here.
```

File: docs/en/week04/practicum04.sbv [hunk 9]
```diff
@@ -1,379 +1,379 @@
...
-here x1 and now you have x2 right so how
-do you express this scalar product here
+here x 1 and now you have x 2, right? so how
+do you express this scalar product here?
 
 0:09:19.980,0:09:26.730
-okay so I'm just writing let me know if
-you took clear so I'm gonna write here a
+okay, so I'm just writing, let me know if
+you it's all clear, so I'm gonna write here:
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:329]:
```suggestion
it's all clear, so I'm gonna write here: 
```

File: docs/en/week04/practicum04.sbv [hunk 10]
```diff
@@ -1,379 +1,379 @@
...
 0:10:20.770,0:10:30.010
-else the same right now wait what what
-is X 1 same thing right different
+else. The same right? No, wait, what? what
+is x 1? same thing, right, different
 
 0:10:30.010,0:10:37.450
-letters so someone say something you're
-following you're completely confused
+letters. so someone say something You're
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:360]:
```suggestion
letters. so someone say something. You're
```

File: docs/en/week04/practicum04.sbv [hunk 11]
```diff
@@ -1,379 +1,379 @@
...
+second term? This one is gonna be x
 
 0:10:53.770,0:11:00.430
-here write x cos x I right and then you
+here, right? times cos xi right, and then you
 had the second term which is going to be
 
 0:11:00.430,0:11:13.920
-what magnitude of a shout I can hear ok
+what? magnitude of a... shout, I can hear. Ok
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:376]:
```suggestion
what? magnitude of a... shout, I can't hear. Ok
```

File: docs/en/week04/practicum04.sbv [hunk 12]
```diff
@@ -469,27 +469,27 @@ product between each column each row of
...
+right? so whenever you have like a linear
 
 0:13:36.220,0:13:40.000
 layer your kernel is gonna be the whole
 row of the matrix and now you see what
 
 0:13:40.000,0:13:47.050
 is the projection of that input on that
-column I am on the input on that flow
+column, I am, on the input on that row
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:480]:
```suggestion
column, oh, damn, on the input on that row
```

File: docs/en/week04/practicum04.sbv [hunk 13]
```diff
@@ -736,154 +736,154 @@ kernel of size three and then here I
 have my data which is gonna be very long
 
 0:20:16.890,0:20:31.169
-right so hold on I can see all right so
-here there are zero then let's say what
+right? and so on. can't draw hold on. I can't see. all right so
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:739]:
```suggestion
right? and so on. Can't draw hold on. I can't see. All right so
```

File: docs/en/week04/practicum04.sbv [hunk 14]
```diff
@@ -736,154 +736,154 @@ kernel of size three and then here I
...
+here there are zero, then let's say what
 
 0:20:31.169,0:20:36.960
 is the other property my my my date my
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:743]:
```suggestion
is the other property my my my data my
```

File: docs/en/week04/practicum04.sbv [hunk 15]
```diff
@@ -736,154 +736,154 @@ kernel of size three and then here I
...
 0:21:07.230,0:21:16.770
-for the rain and I keep going further I
-don't down and I keep going this way all
+futher right? and I keep going further down 
+and I keep going this way all
 
 0:21:16.770,0:21:22.799
-right so what should I put here on the
-bottom what should I put here a zero
+right. so what should I put here on the
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:771]:
```suggestion
right. So what should I put here on the
```

File: docs/en/week04/practicum04.sbv [hunk 16]
```diff
@@ -736,154 +736,154 @@ kernel of size three and then here I
...
+bottom ?what should I put here? a zero,
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:772]:
```suggestion
bottom ? What should I put here? A zero,
```

File: docs/en/week04/practicum04.sbv [hunk 17]
```diff
@@ -1011,210 +1011,210 @@ you that a convolution is just a matrix
...
 i'm going to be importing a library that
 
 0:30:34.270,0:30:39.880
 professor here at NYU made and here I'm
 gonna be just loading my audio data and
 
 0:30:39.880,0:30:43.600
-I'm gonna have that in my X and then my
+I'm gonna have that in my x, and then my
 sampling grade is going to be
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1125]:
```suggestion
sampling rate is going to be
```

File: docs/en/week04/practicum04.sbv [hunk 18]
```diff
@@ -1011,210 +1011,210 @@ you that a convolution is just a matrix
...
 0:31:55.450,0:32:01.240
-like the the the y-axis here is gonna be
-showing you just the amplitude can I
+the y-axis here is gonna be
+showing you just the amplitude. can I
 
 0:32:01.240,0:32:05.530
-turn off the light is okay or you're you
-sure okay thank you
+turn off the light? okay or you're you..?
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1176]:
```suggestion
turn off the light? is it okay? or... are you
```

File: docs/en/week04/practicum04.sbv [hunk 19]
```diff
@@ -1011,210 +1011,210 @@ you that a convolution is just a matrix
...
 
 0:32:50.610,0:32:56.400
-ting ting boom you know you can kind of
+you know you can kind of
 imagine a little bit but okay so what
 
 0:32:56.400,0:33:00.830
-notes did we play there like how can I
+notes did we play there? like how can I
 figure out what put out the notes that
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1213]:
```suggestion
figure out what are the notes that
```

File: docs/en/week04/practicum04.sbv (hunk 4 above)
Comment (mvcaro) [reply docs/en/week04/practicum04.sbv:154]: I was thinking about it as the video progressed and wondering how it was being captioned in other videos. I think a_(1,1) would be more intuitive than a 1 1.

File: docs/en/week04/practicum04.sbv [hunk 20]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 0:38:45.680,0:38:54.860
-the second one the concatenation yeah so
-it's a bit loud I can now cannot even
+the second one, the concatenation, yeah so
+it's a bit loud, I can now cannot even
 
 0:38:54.860,0:39:01.720
-reduce the volume oh I can reduce it
-here too much okay let me go again all
+reduce the volume. O h I can reduce it
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1485]:
```suggestion
reduce the volume. Oh I can reduce it
```

File: docs/en/week04/practicum04.sbv [hunk 21]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 
 0:39:29.360,0:39:36.230
-what so the answer was convolution just
-for the recording I'm asking convolution
+what? so the answer was convolution, just
+for the recording, so I'm asking convolution
 
 0:39:36.230,0:39:43.370
-of what no convolutional spectrogram so
+of what? no convolutional spectrogram, so
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1509]:
```suggestion
of what? not the convolution of the spectrogram, so
```

File: docs/en/week04/practicum04.sbv [hunk 22]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 
 0:41:39.099,0:41:44.019
-the other one one away from his to stop
+the other one, one waveform has to stop
 so it's no longer periodic and whenever
 
 0:41:44.019,0:41:47.609
 you do a Fourier transform of a non
-periodic signal you get you know crop
+periodic signal you get you know crop.
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1600]:
```suggestion
periodic signal you get you know crap.
```

File: docs/en/week04/practicum04.sbv [hunk 23]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
+otherwise you will complain, yeah I can't
+lower the,
 
 0:44:16.800,0:44:23.690
-okay so the first one let's try again
+okay, so the first one, let's try again
 
 0:44:28.880,0:44:37.110
-how cool is that it's just listen to
+*plays sound* how cool is that? it's just listen to
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1718]:
```suggestion
*plays sound* how cool is that? you just listened to
```

File: docs/en/week04/practicum04.sbv [hunk 24]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 convolution okay so basically this was
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1719]:
```suggestion
convolutions, Okay so basically this was
```

File: docs/en/week04/practicum04.sbv [hunk 25]
```diff
@@ -1930,62 +1931,61 @@ dimensional domain which is a two
...
 0:50:07.779,0:50:12.160
-well right
-and so what's gonna be my possible image
+well, right?
+And so what's gonna be my possible image
 
 0:50:12.160,0:50:24.790
-of the X function let's say it's equal 1
-what is it do you know
+of the X function? let's say it's equal 1.
```
Comment (xcastilla) [inline docs/en/week04/practicum04.sbv:1970]:
```suggestion
of the X function? let's say it's c equals 1.
```

**Review by xcastilla — CHANGES_REQUESTED** [review]
Great job! I left a few suggestions for things I caught reviewing your captions while listening to the video.

**Review by xcastilla — APPROVED** [review]
Looks good to me! 
@Atcold  If you have some time, please take a look at this comment before merging to decide what to do: [https://github.com/Atcold/pytorch-Deep-Learning/pull/621#discussion_r494241572](https://github.com/Atcold/pytorch-Deep-Learning/pull/621#discussion_r494241572)

File: docs/en/week04/practicum04.sbv (hunk 4 above)
Comment (Atcold) [reply docs/en/week04/practicum04.sbv]: I agree with a_{1, 1}! Let's update it!

File: docs/en/week04/practicum04.sbv [hunk 26]
```diff
@@ -1282,7 +1282,8 @@ and hear first how it sounds everything.
 So, it's going to be like that.
 
 0:34:14.440,0:34:23.380
-How cool is that?
+How cool is that? 
+*students clap*
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1286]: Hahahahaha! 🤣

File: docs/en/week04/practicum04.sbv [hunk 27]
```diff
@@ -1293,29 +1294,29 @@ This is very nice of you. Let's keep
 going.
 
 0:34:32.360,0:34:36.320
-A♭, then we have an E♭, and then a A♭.
+A flat, then we have E flat and A flat.
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1297]: Nooooo, you're undoing my edits! Hahaha!

File: docs/en/week04/practicum04.sbv [hunk 28]
```diff
@@ -1293,29 +1294,29 @@ This is very nice of you. Let's keep
...
 
 0:34:36.320,0:34:40.380
-The difference between the first A♭and the other one in frequencies
+The difference between the first A flat and the other one in frequencies
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1300]: Same.

File: docs/en/week04/practicum04.sbv [hunk 29]
```diff
@@ -1293,29 +1294,29 @@ This is very nice of you. Let's keep
...
 
 0:34:40.380,0:34:46.540
-is that the first A♭ is going to be twice the frequency as the other one.
+is that the first A is going to be twice the frequency as the other one.
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1303]: These ones I've edited personally.

File: docs/en/week04/practicum04.sbv [hunk 30]
```diff
@@ -1293,29 +1294,29 @@ This is very nice of you. Let's keep
...
 0:34:55.320,0:34:57.680
 On the left hand side, instead, we have the accompagnement,
 
 0:34:57.680,0:35:01.220
-and so we're gonna have a A♭ and B♭
+and so we're gonna have a A flat and B flat
 
 0:35:01.220,0:35:05.620
-And then B♭ and E♭.
+And then B flat and E flat.
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1319]: Please, undo.

File: docs/en/week04/practicum04.sbv [hunk 31]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
 Anyhow, so this was like in the first
 
 0:35:39.290,0:35:43.280
-part you're gonna have the first note
-there's A♭ that you have a B♭
+part you're gonna have the first note,
+there's A flat then you have a B flat
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1353]: Thanks for catching the typo, but let's keep using music symbols.

File: docs/en/week04/practicum04.sbv [hunk 32]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 
 0:35:43.280,0:35:51.550
-A♭ and B so you have … and the and the
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1355]: Same.

File: docs/en/week04/practicum04.sbv [hunk 33]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 
 0:36:02.870,0:36:07.730
-have to a shorter like the half the
-period of the other one right the
+have a shorter like the half the
+period of the other one, right?
 
 0:36:07.730,0:36:12.410
-especially the the a the a flat on top
+especially the A the a flat on top
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1372]: I think I said "the A♭ on top".

File: docs/en/week04/practicum04.sbv [hunk 34]
```diff
@@ -1348,397 +1349,397 @@ Now I can actually go to sleep, later.
...
 is gonna have a period which is half of
 
 0:36:12.410,0:36:20.000
-the period of the a on the bottom one
-right so you have okay if you go half of
+the period of the A on the bottom one,
```
Comment (Atcold) [inline docs/en/week04/practicum04.sbv:1376]: "A♭"

**Review by Atcold — CHANGES_REQUESTED** [review]
Just a few edits for the music part, which I have already personally edited.

File: docs/en/week04/practicum04.sbv (hunk 27 above)
Comment (mvcaro) [reply docs/en/week04/practicum04.sbv:1297]: Hahah! Sorry! I'll revert this and all the ones below

**Review by Atcold — APPROVED** [review]
Great! Thanks!

**Pull request merged**